wiki/People/Elite_Nominators/en.md

```diff
@@ -0,0 +1,104 @@
+# Elite Nominators
+
+**Elite Nominators** are members of the [Beatmap Nominators](/wiki/People/Beatmap_Nominators) and [Nomination Assessment Team](/wiki/People/Nomination_Assessment_Team) who went above and beyond the call in order to display an exceptional dedication towards the ranking process and the overall health of the mapping community by standing out as some of the best nominators of their times.
+
+Elite Nominators are selected by the Nomination Assessment Team through multiple factors, including but not limited to:
+
+- Amount of [nominations](/wiki/Beatmap_ranking_procedure#nominations)
+- Quality of nominated beatmaps
+- Quality of [modding](/wiki/Modding)
+- Amount of [nomination resets](/wiki/Beatmap_ranking_procedure#nomination-resets) received
+- Amount of [Quality Assurance](/wiki/People/Beatmap_Nominators/General_Information#quality-assurance) checks
+- Other notable contributions to the mapping community (such as tooling, community events, etc.)
+- Other qualities that make the nominator stand out from the rest
+
+Starting from [7 February 2020](https://osu.ppy.sh/home/news/2020-02-07-community-contributors-2019), Elite Nominators are recognised through news posts and are awarded:
+
+- The "Elite Nominator" [user title](/wiki/Community/User_title)
+  - Title lasts until the next Elite Nominators wave and is removed if the user is no longer a Beatmap Nominator
+  - Title increments by 1 for each consecutive year the user is an Elite Nominator
```

Comment:
```suggestion
- The "Elite Nominator" [user title](/wiki/Community/User_title), which lasts for 1 year until the next wave of Elite Nominators or when the user leaves the Beatmap Nominators. In case a user is awarded the title again for consecutive years, the title increments by 1 ("Elite Nominator II", etc.).
```